test/unit/specs/components/Personalization/createComponent.spec.js

```diff
@@ -136,9 +135,29 @@ describe("Personalization", () => {
     expect(viewCache.storeViews).not.toHaveBeenCalled();
   });
   it("should trigger onClickHandler at onClick", () => {
-    build();
     personalizationComponent.lifecycle.onClick({ event });
 
     expect(onClickHandler).toHaveBeenCalled();
   });
+  it("should call showContainers() when a request fails", () => {
+    const useCallbackAggregator = () => {
```

Comment:
I know that this is almost exactly the same as [`createCallbackAggregator.js`](https://github.com/adobe/alloy/blob/b75317552df1deeef5e5f5f113e3b3ecdc37b70b/src/utils/createCallbackAggregator.js#L19), but I didn't want to have production code as a dependency in a test. I'm happy to change it or to move this into the [test helpers folder](https://github.com/adobe/alloy/blob/b75317552df1deeef5e5f5f113e3b3ecdc37b70b/test/unit/helpers) if that's a better place for it.

Reply:
Can you change this to use mocks (jasmine.createSpy)? You can use `calls.argsFor` to get the function that was passed to onRequestFailure.